Automattic/wp-calypso
# [Login] Use withCredentials on the ajax login request so cookies are kept

This PR allows our new endpoint:
- `https://wordpress.com/wp-login.php?action=login-endpoint`

To set cookies from CORS ajax requests so we don't need to post to `/wp-login.php` afterwards.

### Testing Instructions
- Apply patch D5455
- Load this branch locally or go to https://calypso.live/login?branch=try/login-redirect
- Ensure that login (with or without 2FA) still works. Note that 2FA fails if an incorrect code is used.

### Reviews
